fuzzers/034-cmt-pll-pips/Makefile

```diff
@@ -3,13 +3,9 @@ PIP_TYPE?=cmt_top
 PIPLIST_TCL=$(FUZDIR)/cmt_top_upper_t.tcl
 TODO_RE=".*CMT_TOP_[LR]_UPPER_T_PLLE2_CLK(IN1|IN2|FBIN)\.CMT_TOP_[LR]"
 
-ifneq (${XRAY_DATABASE}, zynq7)
 MAKETODO_FLAGS=--sides "r_upper_t,l_upper_t" --pip-type ${PIP_TYPE} --seg-type cmt_top --re $(TODO_RE)
-else
-MAKETODO_FLAGS=--sides "l_upper_t" --pip-type ${PIP_TYPE} --seg-type cmt_top --re $(TODO_RE)
-endif
 
-N = 100
+N = 200
```

Comment:
This seems pretty high, are you sure this is needed?  Should there be better input stimulus rather than generating this many samples?

Reply:
I didn't notice much difference in the runtime of CI. But it's because it tooks around 5 hours already. I had to increase the number of specimen as otherwise I got conflicts between iterations as in Zynq7020 there is only one clock region with `CMT_TOP_R_UPPER` tiles compared to 3 clock regions in Artix.

Reply:
What I mean is that it doesn't make much sense for a N this high, it likely means that there is insufficient stimulus in the `top.py`, and rather than cranking the N higher, we should add better stimulus, either through `route_via` in the `generate.tcl` or modifying probabilities in `top.py`.

Reply:
I understand your point and I would also much rather improve the stimulus which seems satisfactory for artix and kintex, but isn't good enough solely for Zynq. Would it be an option at this point to set `N=200` only for Zynq and open an issue about this to deal with it a bit later?